LocalWiki.php

```diff
@@ -52,7 +52,7 @@ function onBeforePageDisplay( OutputPage $out ) {
 	}
 }
 
-if ( in_array( $wgDBname, [ 'polandballwikisongcontestwiki', 'polandsmallswiki', 'ballmediawiki' ] ) ) {
+if ( in_array( $wgDBname, [ 'polandballwikisongcontestwiki', 'polandsmallswiki', 'ballmediawiki', 'polandballfanonwiki', 'partyballwiki', '402611wiki' ] ) ) {
```

Comment:
Can this array now be split to the next lines, this line is becoming quite long.